src/app/domain/scanner/services/scanner.ts

```diff
@@ -3,20 +3,29 @@ import DocumentScanner, {
   ResponseType
 } from 'react-native-document-scanner-plugin'
 
+import {
+  storeSharedMemory,
+  removeSharedMemory
+} from '/libs/localStore/sharedMemory'
+
 type Base64 = string
 
 export const scanDocument = async (): Promise<Base64 | undefined> => {
   try {
+    removeSharedMemory('mespapiers', 'scanDocument')
```

Comment:
Do we really want to clear this before doing the OS scan (that may produce memory flush too)? Would it be better to do this the later we can?